tensorflow/lite/tools/flatbuffer_utils.py

```diff
@@ -274,6 +281,78 @@ def xxd_output_to_object(input_cc_file):
   return convert_bytearray_to_object(model_bytes)
 
 
+def byte_swap_buffer_content(buffer, chunksize, from_endiness, to_endiness):
+  """"Helper function for byte-swapping the buffers field.
+  """
+  to_swap = [buffer.data[i:i+chunksize] for i in range(
+    0, len(buffer.data), chunksize)]
+  buffer.data = b''.join([int.from_bytes(
+    byteswap, from_endiness).to_bytes(
+      chunksize, to_endiness) for byteswap in to_swap])
+
+
+def byte_swap_tflite_model_obj(model, from_endiness, to_endiness):
+  """"Byte swaps the buffers field in a TFLite model.
+
+  Args:
+    model: TFLite model object of from_endiness format.
+  
```

Comment:
you need to describe every arguments

Reply:
Added description for the remaining arguments. Thanks!